pkg/crc/machine/hyperkit/driver_darwin.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/code-ready/machine/libmachine/drivers"
 )
 
-type hyperkitDriver struct {
+type Driver struct {
```

Comment:
@guillaumerose Does golint also complain about type names like `libvirtDriver` or `hyperkitDriver` ?

Reply:
Yes because the package name is the prefix of the struct name.